src/mock/film.js

```diff
@@ -0,0 +1,67 @@
+import {
+  getRandomInteger
+} from '../utils.js';
```

Comment:
расширение `js` можно не указывать при импортах

```suggestion
} from '../utils';
```